EventFilter/L1TRawToDigi/plugins/implementations_stage2/EMTFBlockSP.cc

```diff
@@ -229,11 +229,12 @@ namespace l1t {
 	// if      (nDelay[2]                         >= 1) trk_delay = 2;
 	// else if (nDelay[2] + nDelay[1]             >= 1) trk_delay = 1;
 	// else if (nDelay[2] + nDelay[1] + nDelay[0] >= 1) trk_delay = 0;
-	
+
 	int St_hits[4] = {0, 0, 0, 0}; // Number of matched hits in each station
 
 	for (uint iHit = 0; iHit < res_hit->size(); iHit++) {
```

Comment:
Unpacker has already been pulled into CMSSW_9_3_X - would prefer not to make any changes in l1t-integration right now.